trieste/acquisition/function.py

```diff
@@ -157,6 +158,133 @@ def expected_improvement(model: ProbabilisticModel, eta: TensorType, at: TensorT
     return (eta - mean) * normal.cdf(eta) + variance * normal.prob(eta)
 
 
+class MinValueEntropySearch(SingleModelAcquisitionBuilder):
+    """
+    Builder for the min-value entropy search acquisition function (an adapted
+    version of max-value entropy search suitiable for function minimisation tasks).
+    """
+
+    def __init__(self, search_space: SearchSpace, num_samples: int = 10, grid_size: int = 5000):
+        """
+        :param search_space: The global search space over which the optimisation is defined.
+        :param num_samples: Number of sample draws of the minimal value.
+        :param grid_size: Size of random grid used to fit the gumbel distribution
+            (recommend scaling with search space dimension).
+        """
+        self._search_space = search_space
+
+        if num_samples <= 0:
+            raise ValueError(f"num_samples must be positive, got {num_samples}")
+        self._num_samples = num_samples
+
+        if grid_size <= 0:
+            raise ValueError(f"grid_size must be positive, got {grid_size}")
+        self._grid_size = grid_size
+
+        self.gumbel_samples = None
```

Comment:
is this for testing or do users need access to it?

Reply:
ok. I reckon that as a feature on the public API, this would be scope creep, so can we leave it for future work?

Reply:
Sure. Although this will mean I need to remove some of the tests

Reply:
that's ok by me. Do you not reckon?

Reply:
Okay doke. Will remove